### DISCUSSION TITLE
Alert Flex Direction

### DISCUSSION THREAD

**Discussion by sephclt** [saadeghi/daisyui#1603] [Q&A]
Does the Alert component have Flex direction column by default? I just found it weird cause looking at the documents for Alert it seemed to me that it had Flex direction row, but after trying it I saw it had flex column instead so my button was underneath the alert text.

**Comment by saadeghi** [comment]
It's a row on large screens and column on smaller screen. Because we need items (content, buttons, etc) to stay next to each other but on a small screen they won't fit so it should be column. You can see how it changes on examples: https://daisyui.com/components/alert/
